src/dune

```diff
@@ -1,7 +1,7 @@
 (library
  (name quickchick_plugin)
  (public_name quickchick.plugin)
- (flags :standard -rectypes -w -8-27)
+ (flags :standard -rectypes -warn-error -3 -w -8-27+40)
```

Comment:
Is this warning change OK?

Reply:
I guess so, otherwise the Dune build does fail with any deprecation warning, which the current tree did have.

Reply:
I mean are quickchick devs OK with adding warning 40?

Reply:
Oh, I thought you were talking about warning 3, I guess it makes sense to add warning 40 too as this is the default in coq_makefile.

Reply:
I'm fine with that.